### PR TITLE
BE: remove content of questions for metadata requests

### PR DESCRIPTION
**Pull request by Integraluminium**

### CODE REVIEW

**Review by Jozys — DISMISSED** [review]
Looks good

**Comment by Jozys** [comment]
But why is this still a draft? Is there something missing?

**Comment by Integraluminium** [comment]
It was missing the merge of the other branches, because there were some difficult merge conflicts.
In Addition there were missing some minor changes and the test of everything together.
For example the implementation of favourites required the codebase of the other PR.
That's why it was draft

File: backend/src/requests/user-content-request/user-content-request.service.ts [hunk 1]
```diff
@@ -50,34 +55,32 @@ export class UserContentRequestService {
...
   }
 
-  async getQuestionsOfUser(
+  /**
+   * Get Questions of the user in database format
+   * @param userId
+   * @param sortOptions
+   * @private
+   */
+  private async getQuestionsOfUser(
```
Comment (Jozys) [inline backend/src/requests/user-content-request/user-content-request.service.ts:76]: Is this wrapper method needed?

As the method `myQuestion` could also make the same call of the database service and the sortOptions for that could also be created there.

**Review by Jozys — COMMENTED** [review]
One suggestion from my side. The rest looks fine.

**Review by Jozys — APPROVED** [review]
Looks good.

File: backend/src/requests/user-content-request/user-content-request.service.ts (hunk 1 above)
Comment (Jozys) [reply backend/src/requests/user-content-request/user-content-request.service.ts]: This redundant method was removed within b7a97de

**Pull request merged**